pyiron_base/project/generic.py

```diff
@@ -1514,6 +1513,33 @@ def unpack(self, origin_path, csv_file_name='export.csv', compress=True):
             self, archive_directory=origin_path, df=df, compressed=compress
         )
 
+    @classmethod
+    def register_tools(cls, name: str, creator):
+        """
+        Add a new creator to the project class.
+
+        Example)
```

Comment:
```suggestion
        Example)
        
```
`>>>` have to start with empty line above, otherwise sphinx will mess up the formatting.